src/tests/test_utils.cljs

```diff
@@ -0,0 +1,136 @@
+(ns tests.test-utils
```

Comment:
I think we shouldn't be copy & pasting like this. Just so you know @FFFra, this is a strategy we decided a long time ago to avoid because duplicated code leads to a lot of confusion and debt piles up quickly.

The file `legacy.status-im.utils-test` is only used in a few places, so it's trivial to just move it. This whole namespace is identical except for the lines below:

```
    :restoreAccountAndLogin
    (fn [request]
      (prn native-status)
      (.restoreAccountAndLogin native-status request))
    :loginAccount
    (fn [request] (.loginAccount native-status request))
```

Reply:
Thank you for the context, Icaro. I'm adapting =)

Reply:
Thank you @FFFra 👍🏼

Reply:
Just an FYI @FFFra As soon as you merge this PR I'll adapt it to the new API coming up in https://github.com/status-im/status-mobile/pull/19208. It will be quick for me, and your PR was opened and reviewed way before, so I think yours should be merged and then I'll adapt the test to the new API.